hack/test-go.sh

```diff
@@ -159,11 +161,25 @@ else
     fi
 fi
 
+filtered_test_packages=
+excluded_test_packages=
+for package in ${test_packages}; do
+    if [[ ",${exclude_packages}," =~ ",${package}," ]]; then
```

Comment:
@stevekuznetsov alternatively, I could just make $exclude_packages space separated, then no need for IFS trickery.